app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/utils/BasisOfClaimsExamples.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsreimbursementclaimfrontend.views.utils
+
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.DraftClaim
+
+sealed trait BasisOfClaimsExamples {
+  val items: Seq[Int]
+
+  def buildLabelKey(parentKey: String, index: Int): String
+  def buildTextKey(parentKey: String, index: Int): String
+}
+
+object BasisOfClaimsExamples {
```

Comment:
I'm not big on naming things.
Maybe BasisOfClaimsHint? or BasisOfClaimsHelp?
I had to read through the whole pull request to realise what it is doing...

Reply:
I used the same naming as content designers use. It can be renamed up to you.